### DISCUSSION TITLE
Enter file and insert data

### DISCUSSION THREAD

**Discussion by fellnerse** [nushell/nushell#4223] [Q&A]
From all the examples of the cookbook it looks like you have to open a file with `nuopen or open` then pipe it and add data and save it again.
For me it seems like using `enter` to go inside the nuconfig i.e. and then changing values there feels more natural.
Is this possible? I could not find information on that. And when I try it does not change the data. After inserting/appending its printed, but `ls` will not print it and it's gone.
Also the starship terminal shows a lock icon (🔒) when entering a file.

**Comment by kubouch** [comment]
We're kind of stepping away from using `enter` to step into data in the new engine update. We might support it again at some point but so far we've felt like it hasn't been used or supported much and it can get quite awkward.

I'm not sure how what you describe could work. Is an interesting use case, though, and I think at least theoretically should be possible to implement.

Starship showing lock probably makes sense since you're not in a filesystem anymore, you're in data :-D (that's one example of the awkwardness I mentioned above).
